fabric-datafixer-v1/src/main/java/net/fabricmc/fabric/api/datafixer/SimpleFixes.java

```diff
@@ -0,0 +1,90 @@
+package net.fabricmc.fabric.api.datafixer;
+
+import com.google.common.collect.ImmutableMap;
+import com.mojang.datafixers.DataFixerBuilder;
+import com.mojang.datafixers.Dynamic;
+import com.mojang.datafixers.schemas.Schema;
+import com.mojang.datafixers.util.Pair;
+
+import net.fabricmc.fabric.impl.datafixer.FabricSimpleFixesImpl;
+
+/**
+ * This class file contains several common datafixes most modders would use.
+ */
+public abstract class SimpleFixes {
```

Comment:
Can we just make this an interface given all the fields are public static final and all the methods are public and abstract?